extensions/helix/package.build.ts

```diff
@@ -274,6 +274,225 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
             },
           },
         },
+
+      "space": {
```

Comment:
nit: avoid quotes when possible

```suggestion
      space: {
```

Generally we try to stay consistent within an object, which is why in `items` we use quotes for all keys (since some keys cannot be identifiers)